lib/task-data/schemas/generate-tag.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "rackhd-task-schema.json",
+    "copyright": "Copyright 2016, EMC, Inc.",
+    "title": "Generate Tag Task",
+    "description": "The schema for generate-tag",
+    "describeJob": "Job.Catalog.GenerateTag",
+    "allOf": [
+        { "$ref": "common-task-options.json#/definitions/Options" },
+        {
+            "oneOf": [
```

Comment:
There is similar example, that is [create-megaraid.json](https://github.com/RackHD/on-tasks/blob/master/lib/task-data/schemas/create-megaraid.json#L80), the difference is it uses `anyOf` for `required`, but here should be `oneOf`